test/encrypt_test.dart

```diff
@@ -9,6 +10,27 @@ void main() {
   const text = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit';
   final key = Key.fromUtf8('my32lengthsupersecretnooneknows1');
 
+  group('Fernet', () {
+    final currentDateTime =
+        DateTime.fromMillisecondsSinceEpoch(1565106118 * 1000);
+    final b64key = Key.fromUtf8(base64Url.encode(key.bytes));
```

Comment:
You are getting the UTF-8 key, converting its bytes to 64, then using a 64 bytes hash as UTF-8... Didn't get that part.

Reply:
Fernet specified that the encryption key being a base64, utf8 encoded bytes.
I was translating code from [the python cryptography library](https://github.com/pyca/cryptography/blob/master/src/cryptography/fernet.py#L35).